src/lib/index.js

```diff
@@ -1159,3 +1159,31 @@ lib.fillText = function(calcPt, trace, contOut) {
 lib.isValidTextValue = function(v) {
     return v || v === 0;
 };
+
+lib.formatPercent = function(ratio, n) {
```

Comment:
Interesting. I'm having second-thoughts about my attempt at fixing `tickformat:'p'` in https://github.com/plotly/plotly.js/pull/3868

Reply:
I just closed off https://github.com/plotly/plotly.js/pull/3868 (for now)